net-proxy/3proxy/files/3proxy-0.8.12-gentoo.patch

```diff
@@ -0,0 +1,43 @@
+--- a/Makefile.Linux	2019-03-02 22:34:49.292273238 +0200
++++ b/Makefile.Linux	2019-03-02 22:37:42.735259129 +0200
+@@ -10,11 +10,11 @@
+ BUILDDIR =
+ CC = gcc
```

Comment:
It will not respect user's `$CC`. Probably you need to use `?=` here, and pass `CC="$(tc-getCC)"` to `make`.